CPAC/anat_preproc/anat_preproc.py

```diff
@@ -638,7 +684,8 @@ def acpc_align_head(wf, cfg, strat_pool, pipe_num, opt=None):
      "option_val": "None",
      "inputs": [["desc-preproc_T1w", "desc-reorient_T1w", "T1w"],
                 "T1w_ACPC_template"],
-     "outputs": ["desc-preproc_T1w"]}
+     "outputs": ["desc-preproc_T1w",
+                 "from-affine_to-rigid_mode-image_xfm"]}
```

Comment:
This transform is generated from [ACPC step5](https://github.com/FCP-INDI/C-PAC/blob/feature/abcd_v1.8/CPAC/anat_preproc/utils.py#L4), which converts affine transform to rigid transform. What would you suggest the name for this transform?

Reply:
Oh right, this one is a little tricky- this transform will be used to modify _another_ transform, right? I don't think the BIDS spec has specific language for a case like this but I can double-check. It's possible it can be described in the `mode-*` tag but there is only language for `image`, `surface`, `sphere`, `points` as values for that field.

If this is modifying another transform, what spaces is that transform going from/to? I think it would still be more appropriate to label it that way - and maybe this one can have something like `desc-aff2rig`.

One last small point, in the code you linked, the docstring from FSL describes it as converting from 6 DOF to 12 DOF - usually, in 3D space/images, rigid = 6 and affine = 12, so I wanted to double-check that - unless that FSL code is being used to go in the reverse direction.

Reply:
The transform is from native T1 space to ACPC template space, should we call it `from-T1w_to-template_mode-image_desc-aff2rig_xfm`?

I also find that description a bit weird but here's reference: http://ftp.nmr.mgh.harvard.edu/pub/dist/freesurfer/tutorial_packages/OSX/fsl_501/bin/aff2rigid

Reply:
I would say `from-T1w_to-ACPC` - this is used downstream in the Myelin Map bit right?

Also consider how that looks on the other end - if someone sees the docstring for correct_restore_brain_intensity_ABCD and sees it requires a T1-to-ACPC transform, it will help paint a clearer quick picture of the purpose of that workflow segment as well.